### DISCUSSION TITLE
Wifi password limit?

### DISCUSSION THREAD

**Discussion by S1CKSPEED** [OnionUI/Onion#1010] [Support]
I can't connect to my wifi with my miyoo mini + and latest beta firmware. I think that there is some sort of password limit. My Wifi password length is 63 characters and even I can't see them all when I introduce it in the Miyoo Mini +, I've tried enough times to ensure I am introducing them all. 

Is there a password length limit for Wifi?

Thank you!

**Comment by thomasklemenc** [comment]
Tested this - works for me with 63 characters.

**Comment by S1CKSPEED** [comment]
Well, it has to be some special character or something as I tried thousand times without luck. This is my password btw (couple of numbers changed :p):
39#%xnXJ4%dCR)N5SyzU)mvR*@oUr7cJPad%dCTzARrC2*bY*wgLw%%f5eBp%(%

**Comment by thomasklemenc** [comment]
Well yes, special characters do not work, as stated in the [wiki](https://github.com/OnionUI/Onion/wiki/Frequently-Asked-Questions-%28FAQ%29#my-wifi-wont-connect-as-it-contains-special-characters-what-do-i-do). There you can also find workarounds which you can try.
